src/simplify_algebra.cpp

```diff
@@ -204,7 +204,12 @@ struct find_mul_slice_conv
     }
 };
 
-// a * (x + b) => a * x + a * b
+// ******************************
```

Comment:
we need similar comment on `find_conv_add`.

Reply:
This applies to all the `find_*_add` classes that follow.